wp-content/themes/workingnyc/assets/js/global-9c3ebc0.js

```diff
@@ -1274,6 +1274,19 @@
     googleTranslateElementInit();
   }
 
+ /** Toggle the language dropdown carrot */
+
+  let toggleElement = document.querySelector('[data-js="dialog"]');
+
+  toggleElement.addEventListener('click', (event) => {
+    if(toggleElement.hasAttribute('aria-controls')){
+      let toToggle = document.querySelector('[data-js="language-up-arrow"]');
```

Comment:
- [ ] Use `toggleElement.querySelector` here and in the next line instead of `document.querySelector`
- [ ] Check if toToggle and hideToggle are not null before toggling the up and down arrows